### ISSUE TITLE
Team page is very slow

### ISSUE THREAD

**Issue by g4s8** [zerocracy/farm#1121]
Team page is loading about few seconds. Looks like it happens because we are loading speed, awards, projects, vacatios, rates files for each person in team. We should save some of them in `people.xml` to avoid unnecessary file loading.

**Comment by 0pdd** [comment]
@g4s8 the puzzle [#1123](https://github.com/zerocracy/farm/issues/1123) is still not solved.

**Comment by g4s8** [comment]
@0crat assign @carlosmiranda

**Comment by g4s8** [comment]
@carlosmiranda looks like this bug is related to #841 which is already assigned to you

**Comment by 0crat** [comment]
> @0crat assign @carlosmiranda  ([here](https://github.com/zerocracy/farm/issues/1121#issuecomment-395873288))

@g4s8 The job [#1121](https://github.com/zerocracy/farm/issues/1121) assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1592019689); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
@carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by carlosmiranda** [comment]
@0crat wait, busy on other tasks

**Comment by 0crat** [comment]
> @0crat wait, busy on other tasks ([here](https://github.com/zerocracy/farm/issues/1121#issuecomment-397110385))

@carlosmiranda Can't understand "wait,", try one of these:
  - `assign` Assign a performer to this issue
  - `boost` Set a boost for the job
  - `hello` Just say hello
  - `in` Register this issue in scope (WBS)
  - `out` Close the order and remove this job from scope
  - `pay` Pay a user some extra cash
  - `quality` Review a task
  - `resign` Resign from this issue
  - `status` Check the status of the job
  - `version` Print current version of the bot
  - `wait` Register an impediment for a job

**Comment by g4s8** [comment]
@0crat boost 4x

**Comment by 0crat** [comment]
> @0crat boost 4x ([here](https://github.com/zerocracy/farm/issues/1121#issuecomment-398095410))

@g4s8 Boost 4x was set for [#1121](https://github.com/zerocracy/farm/issues/1121)

**Comment by 0crat** [comment]
Boosting tasks is against our principles, see [§15](http://www.zerocracy.com/policy.html#15): -10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by g4s8** [comment]
@carlosmiranda this task is most urgent, please try to solve it first

**Comment by carlosmiranda** [comment]
@g4s8 noted, will be on it

**Comment by carlosmiranda** [comment]
@g4s8 currently blocked by https://github.com/zerocracy/datum/issues/353

**Comment by carlosmiranda** [comment]
@0crat wait for zerocracy/datum#353

**Comment by 0crat** [comment]
> @0crat wait for zerocracy/datum#353 ([here](https://github.com/zerocracy/farm/issues/1121#issuecomment-398688375))

@carlosmiranda The impediment for [#1121](https://github.com/zerocracy/farm/issues/1121) was registered successfully by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by carlosmiranda** [comment]
@g4s8 see PR #1251 please

**Comment by 0pdd** [comment]
@g4s8 2 puzzles [#1123](https://github.com/zerocracy/farm/issues/1123), [#1267](https://github.com/zerocracy/farm/issues/1267) are still not solved.

**Comment by carlosmiranda** [comment]
@g4s8 can we close this now? PR #1251 merged

**Comment by g4s8** [comment]
@carlosmiranda yes, thanks

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1121](https://github.com/zerocracy/farm/issues/1121) is now out of scope

**Comment by 0crat** [comment]
Order was finished, quality is "good": +68 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by g4s8** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/1121#issuecomment-401034504))

@g4s8 This is what I know about this job in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#1121](https://github.com/zerocracy/farm/issues/1121) is not in scope
  * Job [footprint](https://www.0crat.com/footprint/C3NDPUA8L?q=%7Bjob%3A%27gh%3Azerocracy%2Ffarm%231121%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by 0pdd** [comment]
@g4s8 the puzzle [#1123](https://github.com/zerocracy/farm/issues/1123) is still not solved; solved: [#1267](https://github.com/zerocracy/farm/issues/1267).

**Comment by 0pdd** [comment]
@g4s8 all 2 puzzles are solved here: [#1123](https://github.com/zerocracy/farm/issues/1123), [#1267](https://github.com/zerocracy/farm/issues/1267).
